admin_app/src/components/ContentModals.tsx

```diff
@@ -0,0 +1,144 @@
+// ConfirmDelete.jsx
+import React, { useState, useEffect } from "react";
+import { Content } from "./ContentCard";
+import { XMarkIcon } from "@heroicons/react/20/solid";
+import TextareaAutosize from "react-textarea-autosize";
+
+interface ConfirmDeleteProps {
+  cardToDelete: Content;
+  onDeleteConfirm: (card: Content) => void;
+  onClose: () => void;
+  title: string;
+}
+export const ConfirmDelete: React.FC<ConfirmDeleteProps> = ({
+  cardToDelete,
+  onDeleteConfirm,
+  onClose,
+  title,
+}) => {
+  return (
+    <div onClick={onClose}>
+      <div className="fixed inset-0 z-10 overflow-y-auto flex items-center justify-center">
+        <div className="backdrop-blur bg-opacity-10 absolute inset-0"></div>
+        <div
+          className="bg-gray-200 dark:bg-gray-700 p-4 rounded-lg z-20 shadow-lg dark:outline-none"
+          onClick={(e) => e.stopPropagation()}
+        >
+          <h3
+            className="text-lg dark:text-white leading-6 font-medium text-gray-900"
+            id="modal-title"
+          >
+            {title}
+          </h3>
+          <div className="mt-2">
+            <p className="text-sm text-red-600 ">
+              ⚠️ The action cannot be undone.
```

Comment:
I changed the styles a bit

![image](https://github.com/IDinsight/aaq-core/assets/14125957/58a0c64d-ba93-4ba5-bf46-db2a84f8bc48)